app/src/pages/inside/dashboardPage/dashboardList/dashboardEmptyResults/dashboardEmptyResults.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { GhostButton } from 'components/buttons/ghostButton';
+import styles from './dashboardEmptyResults.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  currentUserDashboardsHeadline: {
+    id: 'DashboardEmptyResults.currentUserDashboardsHeadline',
+    defaultMessage: 'You have no dashboards',
+  },
+  currentUserDashboardsText: {
+    id: 'DashboardEmptyResults.currentUserDashboardsText',
+    defaultMessage: 'Add your first dashboard to analyse statistics',
+  },
+  currentUserDashboardsActionText: {
+    id: 'DashboardEmptyResults.currentUserDashboardsActionText',
+    defaultMessage: 'Add New Dashboard',
+  },
+  sharedDashboardsHeadline: {
+    id: 'DashboardEmptyResults.sharedDashboardsHeadline',
+    defaultMessage: 'No dashboards are shared',
+  },
+});
+
+@injectIntl
+export class DashboardEmptyResults extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    action: PropTypes.func,
+    userDashboards: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    action: () => {},
+    userDashboards: false,
+  };
+
+  render() {
+    const { userDashboards, action, intl } = this.props;
+
+    return (
+      <div className={cx('empty-dashboard')}>
```

Comment:
Component's main class name should be equal to component's name in dash notation.